templates/repo/release/list.tmpl

```diff
@@ -3,16 +3,55 @@
 	{{template "repo/header" .}}
 	<div class="ui container">
 		{{template "base/alert" .}}
-		<h2 class="ui header">
-			{{.i18n.Tr "repo.release.releases"}}
-			{{if .CanCreateRelease}}
-				<div class="ui right">
-					<a class="ui small green button" href="{{$.RepoLink}}/releases/new">
-						{{.i18n.Tr "repo.release.new_release"}}
-					</a>
-				</div>
-			{{end}}
+		<h2 class="ui compact small menu header">
+			<a class="{{if (not .PageIsTagList)}}active{{end}} item" href="{{.RepoLink}}/releases">{{.i18n.Tr "repo.release.releases"}}</a>
+			<a class="{{if .PageIsTagList}}active{{end}} item" href="{{.RepoLink}}/tags">{{.i18n.Tr "repo.release.tags"}}</a>
 		</h2>
+		{{if (and .CanCreateRelease (not .PageIsTagList))}}
+			<a class="ui right small green button" href="{{$.RepoLink}}/releases/new">
+				{{.i18n.Tr "repo.release.new_release"}}
+			</a>
+		{{end}}
+		{{if .PageIsTagList}}
+		<div class="ui divider"></div>
+		<h4 class="ui top attached header">
```

Comment:
The table should be hidden if no tags are present, similar to how releases one is hidden

![chrome_2020-09-15_14-13-29](https://user-images.githubusercontent.com/1447794/93209032-c6f22a80-f75d-11ea-8304-ce039fea098d.png)